examples/math/config.go

```diff
@@ -0,0 +1,47 @@
+package math
```

Comment:
This is nice @rjeczalik. A really different way of exposing configurations. We for example use https://github.com/koding/multiconfig with a pre defined struct. It's more easier and maintainable in the long term. For this PR you don't need to update it as this is just an example :)